python/galois/cpp/libstd/atomic.pxd

```diff
@@ -0,0 +1,58 @@
+cdef extern from "<atomic>" namespace "std" nogil:
```

Comment:
Worth noting: this will be included in a Cython release soon (https://github.com/cython/cython/pull/3469). It appears that's already where you got the file. We can drop this once it's in a new release there.